docs/guides.md

```diff
@@ -523,3 +523,73 @@ Now we enable autostart at the virtual machine level.
 5. Checking the output 
 `# xe vm-param-list uuid=<VM_UUID> | grep other-config`
 
+## Guest UEFI Secure Boot
+
+Enabling UEFI Secure Boot for guests ensures that XCP-ng will ensure that guest VMs only execute trusted binaries.  In practice, these are the binaries released by the operating system (OS) team for the OS in running in the VM (Microsoft Windows, Debian, Alpine, etc.).
+
+### Requirements
+
+* A UEFI guest VM.
+
+### Windows
+
+The steps in this guide are all that are necessary to enable secure boot.  Just follow the instructions provided here.
+
+### Linux
+
+It is required to follow the Linux distribution's instructions for enabling UEFI Secure Boot by installing shim and the distribution's signed binaries.  First follow the instructions from the distribution, then follow the instructions provided here.
+
+### Configure the Host
+
+Before enabling UEFI Secure Boot for guest VMs, first execute the `secureboot-certs` script.  This tool downloads, formats, and installs the publically available certificates from Microsoft.
+
+From the XCP-ng CLI:
+
+```
+# Download and install MS certificates
+secureboot-certs
```

Comment:
Does this work behind a proxy, if the appropriate environment variables are set?

Reply:
IDK.  Since I am doing another version bump anyway, I'll add the ability to use a local cert downloaded manually (avoided adding / retesting due to time constraints) in the case that people can't access the URL from their XCP-ng host or if the MS URL goes down (or changes).

Reply:
As the script is using curl, I think it should be able to download properly if the appropriate environment variable are set by the user (which may be automatic based on their network configuration, I'm not sure).